content/v2.1/en/introduction/features.md

```diff
@@ -18,122 +16,122 @@ The following modules elaborate on the key features and benefits provided by Kub
 
 
 ## Multi-tenant Management
 
-- Multi-tenancy: Provide unified authentication based on fine-grained roles and three-tier authority management.
-- Unified authentication: Support the docking to the centralized authentication system with enterprises based on LDAP/AD protocol. And support single sign-on (SSO) to realize the unified authentication of tenant identity.
-- Authority management: The authority level is divided into three levels, namely, cluster, enterprise space and project. From high to low, we ensure the resource sharing as well as the isolation among different roles at multiple levels. Fully guarantee resource security.
+- Multi-tenancy: Provides unified authentication with fine-grained roles and three-tier authorization system.
+- Unified authentication: Supports docking to a central enterprise authentication system that is LDAP/AD based protocol. And supports single sign-on (SSO) to achieve unified authentication of tenant identity.
+- Authorization system: It is organized into three levels, namely, cluster, workspace and project. We ensure the resource sharing as well as isolation among different roles at multiple levels to fully guarantee resource security.
 
 
 
 ## Multi-dimensional Monitoring
 
-- Fully monitoring dashboard can be operated through a visual interface, while at the same time, the open standard interface is easy to connect with the enterprise operation and maintenance system, so as to unify the operation and maintenance entry to realize centralized operation and maintenance.
-- Three-dimensional second-level monitoring: Instruct the three-dimensional monitoring with second-level frequency and dual-dimensions for 16 indicators.
-- In the cluster resource dimension, we provide multiple indicators such as CPU utilization, memory utilization, CPU load average, disk usage, inode utilization, disk throughput, IOPS, network card rate, container group running state, ETCD monitoring, API Server monitoring.
-- In the dimension of application resources, we provide five monitoring indicators such as CPU consumption, memory consumption, number of container clusters, network outflow rate and network inflow rate for the application. Besides, we support the query by the amount of sorting and custom time range as well as fast location of exceptions.
-- Provide resource usage ranking by node, enterprise space, project, etc.
-- Provide service component monitoring to quickly locate component failures.
+- Monitoring system is fully visualized, and provides open standard APIs for enterprises to integrate their existing operating platforms such as alerting, monitoring, logging etc. in order to have a unified system for their daily operating work.
+- Comprehensive and second precision monitoring metrics: multi-dimensional metrics with second-level precision and more than 16 metrics.
```

Comment:
`Comprehensive and second precision monitoring metrics`

I think `second` should be removed.

Reply:
second is the time, like hour, minute, second.